web/themes/custom/vote_gov/templates/field/double-field-item--field-accordions.twig

```diff
@@ -0,0 +1,14 @@
+{# {% for subitem in item %} #}
+{# {{ dump(item.length)}} #}
+{# {% endfor %} #}
+  {# {% if subitem -%} #}
```

Comment:
These are two  variables you are looking for  
block content id: `{{ elements['#item'].parent.parent.entity.id() }}`  
index id: `{{ elements['#item'].name }}`  

Also you need to update the name of this file so it it `.html.twig` for it to be recognized.